django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1660,6 +1659,10 @@ def get_chants_with_folios(chants_in_feast):
             )
             context["syllabized_text_with_melody"] = text_and_mel
 
+        user = self.request.user
+        context["user_can_proofread_chant"] = user_can_proofread_chants_in_source(
+            user, source
+        )
```

Comment:
the two of us just checked this - we no longer use `user_can_proofread_chants_in_source` anywhere, so this function (in `permissions.py`) should really be renamed `user_can_proofread_chant` and should take (User, Chant) as arguments rather than (User, Source).